init.lua

```diff
@@ -625,6 +630,7 @@ function loot.sellStuff()
     mq.flushevents('Sell')
     if mq.TLO.Window('MerchantWnd').Open() then mq.cmd('/nomodkey /notify MerchantWnd MW_Done_Button leftmouseup') end
     local newTotalPlat = mq.TLO.Me.Platinum() - totalPlat
+    report('Total plat value sold: \ag%s\ax', newTotalPlat)
     loot.logger.Info(string.format('Total plat value sold: \ag%s\ax', newTotalPlat))
```

Comment:
if report includes self in the broadcast then can just remove this log too

Reply:
good point. no need to duplicate the output.